src/main/ingestion-queues/queue.ts

```diff
@@ -49,50 +49,79 @@ export async function startQueue(
     }
 
     try {
+        const redisQueue = startQueueRedis(server, piscina, mergedWorkerMethods)
         if (server.KAFKA_ENABLED) {
             return await startQueueKafka(server, piscina, mergedWorkerMethods)
-        } else {
-            return startQueueRedis(server, piscina, mergedWorkerMethods)
         }
+        return redisQueue
     } catch (error) {
         status.error('💥', 'Failed to start event queue:\n', error)
         throw error
     }
 }
 
-function startQueueRedis(server: Hub, piscina: Piscina | undefined, workerMethods: WorkerMethods): Queue {
+function startQueueRedis(server: Hub, piscina: Piscina, workerMethods: WorkerMethods): Queue {
     const celeryQueue = new CeleryQueue(server.db, server.PLUGINS_CELERY_QUEUE)
 
+    // this queue is for triggering plugin jobs from the PostHog UI
     celeryQueue.register(
-        'posthog.tasks.process_event.process_event_with_plugins',
+        'posthog.tasks.plugins.plugin_job',
         async (
-            distinct_id: string,
-            ip: string | null,
-            site_url: string,
-            data: Record<string, unknown>,
-            team_id: number,
-            now: string,
-            sent_at?: string
+            pluginConfigTeam: Team['id'],
+            pluginConfigId: PluginConfig['id'],
+            type: string,
```

Comment:
This is called `name` in https://github.com/PostHog/posthog/pull/5687, so maybe here as well?

Reply:
Yeah probably the other way around. I wrote the endpoint there first and then here I was trying to be consistent with the terminology in place for jobs. Will update there.

Reply:
I call the variable type over there now, but kept the API as `name` since that's friendlier. "Job type" is confusing